.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?
+- [ ] Have you verified that you are running the latest version of Nancy?
+- [ ] Have you verified if the problem exist in both `debug` and `release` mode?
+- [ ] Have you searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported?
+
+### Description
+[A description of the bug or feature]

```

Comment:
> Nah, I'm just talking about HTML comments. Just wrap it in <!-- *comment here* -->

So the idea would be that it showed up in edit-mode, but was hidden once it was posted? I see your point, but I don't think we should make them optional in that way

> The problem is that a lot of people won't, so we'll get back loads and loads of issues saying "Thanks for reporting the issue to Nancy!" at the bottom 

The whole point of the templates is to try and enforce a higher standard of the reported issues. If we call failure on it before we've even tried, it then what's the point? Also, if we have a template, and people fail to fill it out, we'll simply ask them to go back and update the description or the issue will be closed